relay-server/src/metrics_extraction/event.rs

```diff
@@ -1002,6 +1002,23 @@ mod tests {
                         "resource.render_blocking_status": "blocking"
                     },
                     "hash": "e2fae740cccd3789"
+                },
+                {
+                    "timestamp": 1694732408.3145,
```

Comment:
@jjbayer I ran `cargo insta review` to try to regenerate snapshots, but it didn't seem to produce anything. Could this be because my test case is not fulfilling the proper conditions?